### PR TITLE
Remove Yarn Workspaces

### PR DESCRIPTION
**Pull request by oliverlloyd**
## What does this change?
Now that the work to extract the `guui` and `design` packages out, using `src-*` is complete, our `dotcom-rendering` repository is a monorepo with only 1 package. This doesn't make sense and because we have no intention to add more packages this PR removes the Yarn Workspaces configuration, returning us to a standard yarn repo.

## What work was done
- Removed `private` and `packages` properties from `/package.json`
- Merged `/frontend/package.json` > `/package.json`
- Moved contents of `/frontend` to `src`
- Updated all refs to `packages/frontend` > `src`
- Moved globals in `/index.d.ts` > `/window.guardian.d.ts`
- Moved `/frontend/index.d.ts` > `/index.d.ts`

## Steps to verify
`git checkout oliver/remove-monorepo && make dev`
- [x] `make dev` works
- [x] `make validate` works
- [x] `make cypress` works
- [x] `make percy` works
- [x] `make build` works
- [x] Deploys to code

## Why?
A simple yarn repo is easier for new members of the team to get started on and is generally simpler.

## Link to supporting Trello card
https://trello.com/c/jQeYasfH/827-remove-monorepo

### CODE REVIEW

**Comment by philmcmahon** [comment]
I've looked through this. I think we should be able to fairly easily prove that everything is working by testing in the code environment that e.g. assets get properly updated with the deploy and that code changes are pulled through.

**Comment by nicl** [comment]
@oliverlloyd something I noticed when building locally (`make build` and then `make start`) is that the fonts don't load anymore. I suspect this is just a local issue when running the prod bundle, and we can probably fix this in a separate PR if needed. An example font request that 404'd here and caused this is: http://localhost:9000/static/frontend/fonts/guardian-textsans/noalts-not-hinted/GuardianTextSans-Regular.woff2.

**Comment by tjmw** [comment]
@nicl I took a stab at fixing the serving of static assets when running in production mode locally in commit 04bceb7, and it seems to work. If you get a moment to verify that the changes made in that commit look okay I'd appreciate it! Thanks

File: scripts/frontend/config.js [hunk 1]
```diff
@@ -3,6 +3,7 @@ const { resolve } = require('path');
 const root = resolve(__dirname, '..', '..');
 const dist = resolve(root, 'dist');
 const target = resolve(root, 'target');
+const statik = resolve(root, 'src', 'static');
```
Comment (tjmw) [inline scripts/frontend/config.js]: This is called `statik` because I otherwise get warnings from TypeScript about `static` being a reserved word.

Comment (gtrufitt) [reply scripts/frontend/config.js]: 🤣

**Pull request merged**